lib/permutat.g

```diff
@@ -583,6 +583,34 @@ BIND_GLOBAL( "MappingPermListList", function( src, dst )
 end );
 
 
+# This is a variant of MappingPermListList which only moves points contained
+# in one of src or dst. In GAP 4.9 and higher this method replaces
+# MappingPermListList. In GAP 4.8 it is kept seperate to avoid changing
+# behaviour in a minor release update.
```

Comment:
This comment explains why this function doesn't replace `MappingPermListList`, but fails to say why it is added in the first place (I guess it's so that "code in function ABC can call it to fix bug XYZ")